omnigibson/robots/robot_base.py

```diff
@@ -412,6 +414,14 @@ def remove(self):
 
         # Run super
         super().remove()
+    
+    @property
+    def base_aabb_extent(self):
```

Comment:
I would call this `reset_joint_pos_aabb_extent`. The doc string should say something like "this is the aabb extent of the robot in the robot frame after resetting the joints".